src/index.html

```diff
@@ -11,7 +11,135 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+      >
+      <fieldset class="block-container">
+        <legend>Personal information</legend>
+
+        <label for="surname" class="form-field">Surname:
+          <input type="text" id="surname"
+            name="surname" autocomplete="off"
+          >
```

Comment:
```suggestion
        <label for="surname" class="form-field">
          Surname:
          <input 
            type="text"
            id="surname"
            name="surname"
            autocomplete="off"
          >
```
Форматируй теги таким образом